.github/workflows/connector-ci-tests.yml

```diff
@@ -222,7 +211,9 @@ jobs:
         run: poe install
 
       - name: Fetch connector secrets
-        if: matrix.connector
+        # This will be skipped if the repo or fork doesn't set the GCP_GSM_CREDENTIALS secret.
+        # Later integration tests will likely fail in this case, but we at least let them run.
+        if: matrix.connector && env.GCP_GSM_CREDENTIALS != ''
```

Comment:
Note: secrets fetch will skip if no GCP cred is available. This lets tests at least be run, regardless. Integration tests will likely fail but we at least gather data from unit tests and any other tests that don't actually need secrets.